web-starter/templates/lib/capistrano/tasks/drush.rake

```diff
@@ -127,12 +133,25 @@ namespace :drush do
     end
   end
 
-  desc "Clears the Drupal cache"
-  task :cc do
-    on roles(:db) do
-      within "#{release_path}/#{fetch(:app_webroot, 'public')}" do
-        fetch(:site_url).each do |site|
-          execute :drush, "-y -p -r #{current_path}/#{fetch(:webroot, 'public')} -l #{site}", 'cc all'
+  if fetch(:platform) == "drupal8"
```

Comment:
I think that's correct, we don't need to conditionals around the task configuration.